plugins/inputs/ravendb/ravendb.go

```diff
@@ -0,0 +1,680 @@
+		tags["node_tag"] = databasesResponse.NodeTag
+		if databasesResponse.PublicServerUrl != "" {
+			tags["public_server_url"] = databasesResponse.PublicServerUrl
+		}
+
+		fields := map[string]interface{}{
+			"uptime_in_sec":                               dbResponse.UptimeInSec,
+			"counts_documents":                            dbResponse.Counts.Documents,
+			"counts_revisions":                            dbResponse.Counts.Revisions,
+			"counts_attachments":                          dbResponse.Counts.Attachments,
+			"counts_unique_attachments":                   dbResponse.Counts.UniqueAttachments,
+			"counts_alerts":                               dbResponse.Counts.Alerts,
+			"counts_rehabs":                               dbResponse.Counts.Rehabs,
+			"counts_performance_hints":                    dbResponse.Counts.PerformanceHints,
+			"counts_replication_factor":                   dbResponse.Counts.ReplicationFactor,
+			"statistics_doc_puts_per_sec":                 dbResponse.Statistics.DocPutsPerSec,
+			"statistics_map_index_indexes_per_sec":        dbResponse.Statistics.MapIndexIndexesPerSec,
+			"statistics_map_reduce_index_mapped_per_sec":  dbResponse.Statistics.MapReduceIndexMappedPerSec,
+			"statistics_map_reduce_index_reduced_per_sec": dbResponse.Statistics.MapReduceIndexReducedPerSec,
+			"statistics_requests_per_sec":                 dbResponse.Statistics.RequestsPerSec,
+			"statistics_requests_count":                   dbResponse.Statistics.RequestsCount,
+			"statistics_request_average_duration":         dbResponse.Statistics.RequestAverageDuration,
+			"indexes_count":                               dbResponse.Indexes.Count,
+			"indexes_stale_count":                         dbResponse.Indexes.StaleCount,
+			"indexes_errors_count":                        dbResponse.Indexes.ErrorsCount,
+			"indexes_static_count":                        dbResponse.Indexes.StaticCount,
+			"indexes_auto_count":                          dbResponse.Indexes.AutoCount,
+			"indexes_idle_count":                          dbResponse.Indexes.IdleCount,
+			"indexes_disabled_count":                      dbResponse.Indexes.DisabledCount,
+			"indexes_errored_count":                       dbResponse.Indexes.ErroredCount,
+			"storage_documents_allocated_data_file_in_mb": dbResponse.Storage.DocumentsAllocatedDataFileInMb,
+			"storage_documents_used_data_file_in_mb":      dbResponse.Storage.DocumentsUsedDataFileInMb,
+			"storage_indexes_allocated_data_file_in_mb":   dbResponse.Storage.IndexesAllocatedDataFileInMb,
+			"storage_indexes_used_data_file_in_mb":        dbResponse.Storage.IndexesUsedDataFileInMb,
+			"storage_total_allocated_storage_file_in_mb":  dbResponse.Storage.TotalAllocatedStorageFileInMb,
+			"storage_total_free_space_in_mb":              dbResponse.Storage.TotalFreeSpaceInMb,
+		}
+
+		if dbResponse.TimeSinceLastBackupInSec != nil {
+			fields["time_since_last_backup_in_sec"] = dbResponse.TimeSinceLastBackupInSec
+		}
```

Comment:
Set field directly as `nil` values for fields will be ignored.

Reply:
done

Reply:
Thx, please mark as resolved.